diagnostic.md

```diff
@@ -32,23 +42,27 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
```

Comment:
Yes, but it would be nice to include the arguments for the setters to differentiate them